app/controllers/follow_requests_controller.rb

```diff
@@ -1,5 +1,7 @@
 class FollowRequestsController < ApplicationController
   before_action :set_follow_request, only: %i[ show edit update destroy ]
+  before_action { authorize @follow_request || FollowRequest }
```

Comment:
good. you can use similar before action to make your comments controller more DRY